packages/iris-grid/src/IrisGridProxyModel.ts

```diff
@@ -476,6 +493,74 @@ class IrisGridProxyModel extends IrisGridModel {
     this.model.filter = filter;
   }
 
+  get partition(): readonly unknown[] {
+    return this.originalModel.partition;
+  }
+
+  set partition(partition: readonly unknown[]) {
+    log.debug('set partition', partition);
+    this.originalModel.partition = partition;
```

Comment:
I don't really like this, where you're mutating the original underlying model to get the next model... instead should just be creating the next model here and calling `setNextModel`.